models/sipnet/R/write.configs.SIPNET.R

```diff
@@ -360,6 +323,59 @@ write.config.SIPNET <- function(defaults, trait.values, settings, run.id, inputs
   }  ## end loop over PFTS
   ####### end parameter update
   
+  #### write INITIAL CONDITIONS here ####
+  IC.path = settings$run$inputs$poolinitcond$path
+  if (!is.null(IC.path)) {
+    
+    IC.nc = ncdf4::nc_open(IC.path) #check for successful open?
+    
+    ## plantWoodInit gC/m2
+    plantWood = try(ncdf4::ncvar_get(IC.nc,"AbvGrndWood"),silent = TRUE)
+    if (!is.na(plantWood) && is.numeric(plantWood)) {
+      param[which(param[, 1] == "plantWoodInit"), 2] <- plantWood * 1000 #MsTMIP AbvGrndWood kgC/m2
+    }
+    else{
+      #try back-calculate from LAI,sla, and total biomass? where is total biomass?
+    }
+    ## laiInit m2/m2
+    lai = try(ncdf4::ncvar_get(IC.nc,"LAI"),silent = TRUE)
+    if (!is.na(lai) && is.numeric(lai)) {
+      param[which(param[, 1] == "laiInit"), 2] <- lai
+    }
+    ## litterInit gC/m2
+    litter = try(ncdf4::ncvar_get(IC.nc,"litter_carbon_content"),silent = TRUE)
+    if (!is.na(litter) && is.numeric(litter)) {
+      param[which(param[, 1] == "litterInit"), 2] <- litter * 1000 #CF litter_carbon_content kg/m2
+    }
+    ## soilInit gC/m2
+    soil = try(ncdf4::ncvar_get(IC.nc,"soil_carbon_content_of_soil_layer"),silent = TRUE)
+    if (!is.na(soil) && is.numeric(soil)) {
+      param[which(param[, 1] == "soilInit"), 2] <- sum(soil) * 1000 #MsTMIP TotSoilCarb kg C/m2
+    }
+    ## soilWFracInit fraction
+    soilWFrac = try(ncdf4::ncvar_get(IC.nc,"SoilMoistFrac"),silent = TRUE)
+    if (!is.na(soilWFrac) && is.numeric(soilWFrac)) {
+      param[which(param[, 1] == "soilWFracInit"), 2] <- sum(soilWFrac)
+    }
+    ## litterWFracInit fraction
+    litterWFrac = soilWFrac
+    ## snowInit cm water equivalent
+    snow = try(ncdf4::ncvar_get(IC.nc,"SWE"),silent = TRUE)
+    if (!is.na(snow) && is.numeric(snow)) {
+      param[which(param[, 1] == "snowInit"), 2] <- snow #MsTMIP SWE kg/m2, need to convert
+    }
```

Comment:
Not sure how to handle units for this without a density measurement; this may need to change